Cargo.toml

```diff
@@ -3,7 +3,7 @@ members = ["rcgen", "rustls-cert-gen"]
 resolver = "2"
 
 [workspace.dependencies]
-aws-lc-rs = "1.6.0"
+aws-lc-rs = { version = "1.6.0", default-features = false }
```

Comment:
I think this wants `features = ["aws-lc-sys"]`, like in [rustls](https://github.com/rustls/rustls/blob/2615a09d6d3e8ecff841306ed43069071369d1fe/rustls/Cargo.toml#L20C77-L20C103), to fix the CI failure.

Reply:
Continuation in https://github.com/rustls/rcgen/pull/286#discussion_r1697580115.